Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.yml

```diff
@@ -2420,89 +2420,1435 @@ script:
+      type: String
+    - contextPath: PrismaCloud.AssetVulnerabilities.problem.parameters
+      description: 'The parameters of the problem.'
+      type: Unknown
+    - contextPath: PrismaCloud.AssetVulnerabilities.accountId
+      description: 'The account ID of the asset vulnerability.'
+      type: String
+    - contextPath: PrismaCloud.AssetVulnerabilities.count
+      description: 'The count of asset vulnerabilities.'
+      type: String
+    - contextPath: PrismaCloud.AssetVulnerabilities.createdOn
+      description: 'The created on of the asset vulnerability.'
+      type: Number
+    - contextPath: PrismaCloud.AssetVulnerabilities.customerId
+      description: 'The customer ID of the asset vulnerability.'
+      type: Number
+    - contextPath: PrismaCloud.AssetVulnerabilities.cveId
+      description: 'The CVE ID of the asset vulnerability.'
+      type: String
+    - contextPath: PrismaCloud.AssetVulnerabilities.description
+      description: 'The description of the asset vulnerability.'
+      type: String
+    - contextPath: PrismaCloud.AssetVulnerabilities.externalId
+      description: 'The external ID of the asset vulnerability.'
+      type: Number
+    - contextPath: PrismaCloud.AssetVulnerabilities.id
+      description: 'The ID of the asset vulnerability.'
+      type: String
+    - contextPath: PrismaCloud.AssetVulnerabilities.normalizedName
+      description: 'The normalized name of the asset vulnerability.'
+      type: String
+    - contextPath: PrismaCloud.AssetVulnerabilities.normalizedNames
+      description: 'The normalized names of the asset vulnerability.'
+      type: String
+    - contextPath: PrismaCloud.AssetVulnerabilities.nvdUrl
+      description: 'The nvd URL of the asset vulnerability.'
+      type: String
+    - contextPath: PrismaCloud.AssetVulnerabilities.link
+      description: 'The link of the asset vulnerability.'
+      type: String
+    - contextPath: PrismaCloud.AssetVulnerabilities.rawData
+      description: 'The raw data of the asset vulnerability.'
+      type: String
+    - contextPath: PrismaCloud.AssetVulnerabilities.regionId
+      description: 'The region ID of the asset vulnerability.'
+      type: String
+    - contextPath: PrismaCloud.AssetVulnerabilities.resourceCloudId
+      description: 'The resource cloud ID of the asset vulnerability.'
+      type: String
+    - contextPath: PrismaCloud.AssetVulnerabilities.resourceId
+      description: 'The resource ID of the asset vulnerability.'
+      type: Number
+    - contextPath: PrismaCloud.AssetVulnerabilities.resourceUrl
+      description: 'The resource URL of the asset vulnerability.'
+      type: String
+    - contextPath: PrismaCloud.AssetVulnerabilities.riskFactors
+      description: 'The risk factors of the asset vulnerability.'
+      type: String
+    - contextPath: PrismaCloud.AssetVulnerabilities.rlUpdatedOn
+      description: 'The rl updated on of the asset vulnerability.'
```

Comment:
What is rl?

Is this a date?  A version number?
The date on which the rl was updated?